### ISSUE TITLE
Crops

### ISSUE THREAD

**Issue by TheCyberBrick** [Angry-Pixel/The-Betweenlands#204]
Aspect Crop: Lets the player produce low amounts of certain aspects. The player has to combine an infusion with plant tonic and then give that plant tonic to this crop. Once the crop is grown its fruits will have the aspects that were in the infusion. The more aspects that infusion has the harder it is to grow the plant or the faster it will decay.

Glueshroom: Drops edible glue

**Comment by Sevenoon** [comment]
Hello, yes, I just about stumbled upon this place, and I happen to of read this feature. Not sure if you already have a texture or not, or if this is scrapped, but I made some textures of the Aspect Fruit. I made 3 just in case you dislike one, or like another. I don't expect for them to be used, but decided to quickly make them. All textures were made legitimately from scratch.

The Imgur album can be found here:
http://i.imgur.com/miDZVS9.png
